bitnami/redis/values.yaml

```diff
@@ -138,6 +138,59 @@ sentinel:
     labels: {}
     loadBalancerIP:
 
+  ## Prometheus Exporter / Metrics for Redis Sentinel
+  ##
+  metrics:
+    enabled: false
+
+    image:
+      registry: docker.io
+      repository: leominov/redis_sentinel_exporter
```

Comment:
As it's mentioned in the [Contributing Guideline](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#adding-a-new-chart-to-the-repository):

> The chart should use Bitnami based container images. If they don't exist, you can open a GitHub issue and we will work together to create them.

I am afraid that we cannot accept this PR using a third-party image.

Reply:
Updated to use `bitnami/redis-sentinel-exporter`